dandi/validate_types.py

```diff
@@ -1,35 +1,97 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from enum import Enum
+from enum import Enum, IntEnum, auto, unique
 from pathlib import Path
+from typing import Any
+
+from dandi.utils import StrEnum
+
+
+@unique
+class Standard(StrEnum):
+    """Standards to validate against"""
+
+    BIDS = auto()
+    DANDI_LAYOUT = "DANDI-LAYOUT"
+    DANDI_SCHEMA = "DANDI-SCHEMA"
+    HED = auto()
+    NWB = auto()
+    OME_ZARR = "OME-ZARR"
+    ZARR = auto()
+
+    # File formats (For denoting validation failures in file format level)
+    JSON = auto()
+    TSV = auto()
+    YAML = auto()
+
+
+@unique
+class Validator(StrEnum):
+    """Validators that are used to do validation"""
+
+    bidsschematools = auto()
```

Comment:
now soon we would have `bids_validator_deno` or alike 

and we could provision also for `hed_python_validator`